test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -731,6 +731,7 @@ protected static void wipeDataStreams() throws IOException {
                     String name = (String) snapshotInfo.get("snapshot");
                     if (SnapshotState.valueOf((String) snapshotInfo.get("state")).completed() == false) {
                         inProgressSnapshots.computeIfAbsent(repoName, key -> new ArrayList<>()).add(snapshotInfo);
+                        logger.info("snapshot [{}/{}]] is not complete. snapshot info [{}]", repoName, name, snapshotInfo);
```

Comment:
How does this help us when the problem in the linked test seems to be that we're leaking a restore that prevents a snapshot from being deleted? We know which snapshot we fail to delete but not why we leaked the restore. This doesn't seem to get us any new information?